backend/routes/event_payment_routes/admin_refund_routes.py

```
@@ -0,0 +1,940 @@
+    validate_refund_amount, 
+    calculate_refund_options, 
+    get_refund_summary
+)
+from models.transaction import RefundStatus, Transaction, RegistrationPaymentStatus, PaymentStatus
+from helpers.paypalHelper import process_paypal_refund
+from helpers.RefundEmailHelper import (
+    send_refund_approved_notification,
+    send_refund_completed_notification,
+    send_refund_rejected_notification
+)
+import logging
```

Ruff (0.14.2)

Redefinition of unused `logging` from line 4

Remove definition: `logging`

(F811)